validator/sawtooth_validator/consensus/proxy.py

```diff
@@ -18,11 +18,16 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class UnknownBlock(Exception):
```

Comment:
This exception already exists in the SDK: https://github.com/hyperledger/sawtooth-core/blob/b66c6d0f47c323efc3a3e6d81f9a0a42d7070733/sdk/python/sawtooth_sdk/consensus/exceptions.py#L25